NOTICES.txt

```diff
@@ -0,0 +1,67 @@
+Notices and Licenses file

```

Comment:
We have a License.txt. Could we put this there? If we can do that, we do not have to change the package generation.
If no, do we still need the Licnese.txt

Reply:
You still need the license.txt and the notices. both need to be in the package.

Reply:
Notices.txt will cover licenses for the all the other open source projects that we use in the library. The license is the main license for the code we contribute.